packages/fuels/Forc.toml

```diff
@@ -101,3 +101,6 @@ members = [
   'tests/types/scripts/script_u256',
   'tests/types/scripts/script_vectors',
 ]
+
+[patch.'https://github.com/fuellabs/sway']
+std = { git = "https://github.com/fuellabs/sway", branch = "hal3e/fuel-core-master2" }
```

Comment:
Do we want to track updating this and the other pathes this with an issue?

Reply:
https://github.com/FuelLabs/fuels-rs/issues/1201